client/Components/Header.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Header = () => {
```

Comment:
Don't think you need this, can just put it AnalystRatings class or in a base html file